modules/configuring-maximum-revision-timeout.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: PROCEDURE
+[id="configuring-maximum-revision-timeout_{context}"]
+= Configuring maximum revision timeout
+
+You can set the maximum allowed revision timeout to ensure that no revision can exceed a specific limit.
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} and Knative Serving.
+* You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
+
+.Procedure
+
+* To configure the maximum revision timeout, set the `max-revision-timeout-seconds` field in the `KnativeServing` custom resource (CR):
++
+[source,yaml]
+----
+apiVersion: operator.knative.dev/v1beta1
```

Comment:
This should be:
```
apiVersion: operator.knative.dev/v1beta1
kind: KnativeServing
metadata:
  name: knative-serving
  namespace: knative-serving
spec:
  config:
    defaults:
      max-revision-timeout-seconds: "600"
```

Reply:
... The KnativeServing custom resource has a little different structure than upstream ConfigMaps.